### PR TITLE
RSX exception fix

### PR DESCRIPTION
**Pull request by Nekotekina**
VBlank thread management fix

### CODE REVIEW

**Comment by raven02** [comment]
Tested with Disgaea 3 , it show up the error in console though resume didn't help the game to progress.

**Comment by Nekotekina** [comment]
Why would it progress?

**Comment by raven02** [comment]
As i assume it is a bogus command and resume can skip it .

File: rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp [hunk 1]
```diff
@@ -278,6 +278,7 @@ D3D12GSRender::~D3D12GSRender()
 
 void D3D12GSRender::on_exit()
 {
+	return GSRender::on_exit();
```
Comment (danilaml) [inline rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp]: `return` in `void` function?

Comment (Nekotekina) [inline rpcs3/Emu/RSX/D3D12/D3D12GSRender.cpp]: It must clarify this is the last (tail) call and other actions must be placed before.

**Pull request merged**